rules.py

```diff
@@ -86,6 +86,9 @@ def arg_parser():
     parser_validate.add_argument(
         "election_file", help="XML election file to be validated",
         metavar="election_file", type=lambda x: validate_file(parser, x))
+    parser_validate.add_argument(
+        "-l", "--lf", help="Local ocd-id csv file path", required=False,
```

Comment:
Nit: These option names seem a bit vague/confusing. I'd just go with

--ocdid-file

and no short options.